src/main/java/ToDo.java

```diff
@@ -0,0 +1,8 @@
+public class ToDo extends Task
```

Comment:
Perhaps you could check if it is supposed to be spelled todo or to do. I believe it might be more readable if it is read as one word

Reply:
I believe to do is more suitable here, thank you nontheless.